Edge10.CouchDb.Client/CouchApi.cs

```diff
@@ -42,12 +44,15 @@ public CouchApi(ICouchDbConnectionStringBuilder connectionString) : this(connect
 		/// </summary>
 		/// <param name="connectionString">The connection string.</param>
 		/// <param name="httpClient">The HTTP client facade.</param>
-		internal CouchApi(ICouchDbConnectionStringBuilder connectionString, IHttpClientFacade httpClient)
+		/// <param name="eventLog">The couch event log.</param>
+		internal CouchApi(ICouchDbConnectionStringBuilder connectionString, IHttpClientFacade httpClient, ICouchEventLog eventLog)
```

Comment:
Yeah, the idea to wrap HttpClient for testing but keep it internal so I use this constructor for testing only